### PR TITLE
added API calls to the login, phone number and selfie pages

### PR DESCRIPTION
**Pull request by adelakuntobi**

### CODE REVIEW

File: assets/js/app.js [hunk 1]
```diff
@@ -4,7 +4,7 @@ const callLoader = () => {
 const removeLoader = () => {
   $(".loader").remove()
 }
-
+const BASE_URL = "https://makecards-bd-ulrhs.ondigitalocean.app/api/fingerprint/"
```
Comment (Yigaue) [inline assets/js/app.js:7]: You know you can use .env in Js file right?

File: assets/js/app.js [hunk 2]
```diff
@@ -19,12 +19,45 @@ const inputValidator = (inputID, name) => {
...
-document.querySelectorAll('input[type="range"]').forEach((input) => { 
+document.querySelectorAll('input[type="range"]').forEach((input) => {
   input.draggable = true;
   input.addEventListener('dragstart', stop);
-});
\ No newline at end of file
+});
+
+const apiCall = async (method, url, raw) => {
+  const access_id = "ff8af6fb-b1bf-4434-9c1c-c156898a564a"
```
Comment (Yigaue) [inline assets/js/app.js:33]: Here too.

File: login.html [hunk 3]
```diff
@@ -98,29 +100,29 @@ <h2 class="heading">Fingerprint Enrolment</h2>
...
-                duration: 7000,
-                close: true,
-                stopOnFocus: true,
-              }).showToast();
-            }
-          }, 3400);
+          // callLoader();
+
+          const res = await apiCall("GET", url + pin);
+          console.log(res);
```
Comment (Yigaue) [inline login.html:106]: Don't forget to remove this log.

File: phone.html [hunk 4]
```diff
@@ -107,26 +107,37 @@ <h2 class="heading mb-5">Fingerprint</h2>
...
-              // $("#selfieContainer").removeClass("hidden")
-              // $("#phoneNumber").addClass("hidden")
+              } else {
+              }
+              // console.log(res);
+            }
+          } else {
+            const res = await apiCall("POST", "authenticate", {
+              reference,
+              phone_number: $("#phoneNo").val().substring(1),
```
Comment (Yigaue) [inline phone.html:133]: consider 080, +234, 234, 803(without the first zero) and validate accordingly.

File: selfie.html [hunk 5]
```diff
@@ -58,17 +61,33 @@
...
+            style="display: none"
+            alt=""
+            class="w-64 h-64 object-cover mx-auto block"
+          />
         </div>
+        <p
+          id="helperText"
+          class="block w-fit bg-gray-200 text-xs py-2.5 px-2 rounded-3xl mt-4 font-bold mx-auto text-center"
+        >
+          Please fit in your face into the frame Ears must show
```
Comment (Yigaue) [inline selfie.html:76]: *ears not Ears.

File: assets/js/app.js (hunk 2 above)
Comment (adelakuntobi) [reply assets/js/app.js:33]: For this access id, the application is suppose to be connected to an external page where the users have to request as a company, but then the UI is not ready and I don't want to wait for that as a blocker so I hardcoded it to be able to move forward

**Pull request merged**